doc/source/code/Utilities.rst

```diff
@@ -8,10 +8,55 @@ Apart from the :doc:`operator <Operators>` classes, `eko` also provides some uti
   -  Implementation of the flavor number scheme and the quark thresholds both for
      the :class:`eko.strong_coupling.StrongCoupling` and the :doc:`operators <../theory/Matching>`
 
+  .. include:: IO-tabs/ThresholdConfig.rst
 
-.. include:: IO-tabs/ThresholdConfig.rst
+  When running in |VFNS| it is important to specify the number of flavors active at each given scale, since the evolution path
+  can be different depending of the chosen setting. This path is determined by :meth:`eko.thresholds.ThresholdsAtlas.path`.
 
+  Let us consider an example to better illustrate how it works.
+  Imagine to have a boundary condition ``q2_ref=64, nf_ref=3`` and would like to evolve your object (|PDF| or :math:`a_s`) to a lower
+  scale (say ``Q=7``) but giving different number of active flavors to the final state and having set the heavy quarks mass thresholds
```

Comment:
let's stick to our default choice, that is we speak in Q2 and *not* Q (you can do `Q2=7**2` ...) - and actually why don't you go for `math:` - we speak about math here ...

Reply:
I thought it was more clear to follow the same syntax we use in the code since we are referring directly to our implementation. I have fixed the squared, which were not consistent.